cyclonedx_py/parser/pipenv.py

```diff
@@ -43,14 +51,25 @@ def __init__(
         debug_message('loading pipenv_contents')
         pipfile_lock_contents = json.loads(pipenv_contents)
         pipfile_default: Dict[str, Dict[str, Any]] = pipfile_lock_contents.get('default') or {}
+        self._process_items(pipfile_default, 'default', use_purl_bom_ref, debug_message=debug_message)
+
+        if OmitCategory.DEV not in omit_category:
+            pipfile_develop: Dict[str, Dict[str, Any]] = pipfile_lock_contents.get('develop') or {}
+            self._process_items(pipfile_develop, 'develop', use_purl_bom_ref, debug_message=debug_message)
 
+    def _process_items(self, items: Dict[str, Dict[str, Any]], group: str,
+                       use_purl_bom_ref: bool,
+                       debug_message: DebugMessageCallback) -> None:
         debug_message('processing pipfile_default')
-        for (package_name, package_data) in pipfile_default.items():
+        for (package_name, package_data) in items.items():
             debug_message('processing package: {!r} {!r}', package_name, package_data)
             version = str(package_data.get('version') or 'unknown').lstrip('=')
             purl = PackageURL(type='pypi', name=package_name, version=version)
             bom_ref = purl.to_string() if use_purl_bom_ref else None
             c = Component(name=package_name, bom_ref=bom_ref, version=version, purl=purl)
+            c.properties.add(Property(
+                name='cdx:pipenv:package:category',
```

Comment:
The documented property is `cdx:poetry:package:group` as per https://github.com/CycloneDX/cyclonedx-property-taxonomy/blob/main/cdx/poetry.md or is this a new property?